app/javascript/packs/Feed.jsx.erb

```diff
@@ -0,0 +1,96 @@
+import { h, Component } from 'preact';
+import PropTypes from 'prop-types';
+
+<% helpers = ActionController::Base.helpers %>
+
+const FEED_ICONS = {
+  COMMENTS_ICON: '<%= helpers.image_path("comments-bubble.png") %>',
```

Comment:
This needs to be an ERB template so that we can generate Fastly URLs for these icons, when in PROD.